src/main/resources/swagger/api-docs.yaml

```diff
@@ -4106,6 +4106,47 @@ paths:
         - Snapshots
       summary: Add a reference to a snapshot in the Terra Data Repo
       operationId: createDataRepoSnapshot
+    get:
+      x-passthrough: true
+      x-passthrough-target: rawls
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/DataRepoSnapshotList'
+        '404':
+          description: Workspace not found or user lacks permissions
+          schema:
+            $ref: '#/definitions/ErrorReport'
+        '500':
+          description: Internal Error
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      parameters:
+      - in: path
+        description: The workspace namespace
+        name: namespace
+        required: true
+        type: string
+      - in: path
+        description: The workspace name
+        name: name
+        required: true
+        type: string
```

Comment:
for namespace and name, use:
```
      parameters:
        - $ref: '#/parameters/workspaceNamespaceParam'
        - $ref: '#/parameters/workspaceNameParam'
```
to normalize the params and cut down on lines of code. See the create-snapshot definition just previous to this as an example.